ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/WinMSSQL.py

```diff
@@ -1407,6 +1471,35 @@ def get_ao_oms(self, results, log):
                               al_modname)
                              ].append(al_om)
 
+        # 4. Availability Databases
+        availability_databases = results.get('ao_info', {}).get('availability_databases', {})
+        for adb_id_tuple, adb_info in availability_databases.iteritems():
+            # Get SQL Instance for containing relation
+            adb_owner_id, adb_index = adb_id_tuple
+            if not adb_owner_id or adb_owner_id not in sql_instances:
+                log.warn('Empty owner SQL instance for Availability Database {}. Skipping.'.format(adb_info.get('name'),
+                                                                                                   adb_index))
+                continue
+
+            # add owner SQL Instance related data to databases info dict
+            adb_info['adb_owner_id'] = adb_owner_id
+            adb_info['sql_hostname_fqdn'] = sql_instances[adb_owner_id].get('sql_hostname_fqdn')
+            adb_info['sql_server_name'] = sql_instances[adb_owner_id].get('sql_server_name')
+            adb_info['instancename'] = sql_instances[adb_owner_id].get('instance_original_name')
+
+            # Determine Availability Replica to which database belongs
+            db_replica_id = ag_and_instance_to_replica_mapping.get(
+                (adb_info.get('ag_res_id'), adb_owner_id)
+            )
+            adb_info['db_replica_id'] = db_replica_id
+
+            adb_om = ObjectMap()
+            adb_om = fill_adb_om(adb_om, adb_info, self.prepId)
+            result['oms'][3][(adb_relname,
```

Comment:
May be it is good to replace a magic number `3` with something that provides more information ?